src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -189,14 +189,28 @@ private void RetrieveMatchingProcessesByProcessName()
                 if (!found &&
                     !WildcardPattern.ContainsWildcardCharacters(pattern))
                 {
+                    int x=0;
+                    if(Int32.TryParse(pattern, out x) && x>0){
```

Comment:
This is slightly cleaner syntax if you can do it that way.
```suggestion
                    if(int.TryParse(pattern, out int x) && x>0){
```